From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-migrate-convert-to-folio_isolate_movable-checkpatch-fixes
Date: Tue Aug 27 06:17:16 PM PDT 2024

WARNING: suspect code indent for conditional statements (8, 24)
#24: FILE: mm/migrate.c:199:
 	else
+			isolated = folio_isolate_movable(folio,

total: 0 errors, 1 warnings, 10 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/mm-migrate-convert-to-folio_isolate_movable.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Baolin Wang <baolin.wang@linux.alibaba.com>
Cc: David Hildenbrand <david@redhat.com>
Cc: Kefeng Wang <wangkefeng.wang@huawei.com>
Cc: Matthew Wilcox (Oracle) <willy@infradead.org>
Cc: Zi Yan <ziy@nvidia.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/migrate.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/mm/migrate.c~mm-migrate-convert-to-folio_isolate_movable-checkpatch-fixes
+++ a/mm/migrate.c
@@ -197,8 +197,7 @@ bool isolate_folio_to_list(struct folio
 	if (lru)
 		isolated = folio_isolate_lru(folio);
 	else
-			isolated = folio_isolate_movable(folio,
-							 ISOLATE_UNEVICTABLE);
+		isolated = folio_isolate_movable(folio, ISOLATE_UNEVICTABLE);
 
 	if (!isolated)
 		return false;
_
